app/shared/buttons/BlackOutlinedButton.module.scss

```diff
@@ -0,0 +1,20 @@
+.button {
```

Comment:
do you need this file now that we are not using mui button?

Reply:
yeah we need that